design/treelist/bm.rhm

```diff
@@ -0,0 +1,133 @@
+#lang rhombus
+import:
+  rhombus/measure
+  "treelist.rhm".TreeList
+
+def M = 1000000
+def N = 10
+
+def show_TreeList = #false
+
+macro 'bm $who: $body':
+  'block:
+     print($who +& "                ".substring(0, math.max(0, 16 - ($who).length())))
+     measure.time:
+       ~gc
+       $body
+     #void' 
+
+bm "+ Pair.cons":
+  for values(r = #false) (i: 0..M):
+    for values (l = []) (i: 0..N):
```

Comment:
Style: space after `values`

Reply:
Thanks for the working through this very large set of changes!

This one is hopefully fixed (HF) — that is, fixed unless I still got it wrong in the latest commit.